.github/workflows/propose_translation.yml

```diff
@@ -0,0 +1,58 @@
+name: Propose Translation
+on:
+  workflow_call:
+    inputs:
+      branch:
+        type: string
+        default: ${{ github.ref_name }}
+      language:
+        type: string
+        default: ""
+      locale_folder:
+        type: string
+        default: "locale"
+      reviewer:
+        type: string
+        default: "emphasize"
```

Comment:
Recommend an array instead

Reply:
the [action](https://github.com/peter-evans/create-pull-request) accepts a comma or newline-separated "list"